### DISCUSSION TITLE
Double sided screen

### DISCUSSION THREAD

**Discussion by metingul** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#564] [Q&A]
Hello,
I used P4, 80X40 pixel modules and it works great. I want to make double sided screen, of course I can resize scrren sizes and duplicate content but the screen size is 160X80, so I dont want to workload to ESP, may be if there any solution for double screen/duplicating.

thank you.

**Comment by mrcodetastic** [comment]
Sorry.  It's not clear to me what you are asking.

**Comment by metingul** [comment]
The resolution of the screen I made is 160X80 pixels. If I make it double-sided, I can use it by making the screen resolution 160X160, but I don’t think the ESP will support a 160X160 resolution. Both sides of the screen will display the exact same content, maybe there is a simpler way to display the same content on both sides, I pulled a parallel cable but it didn’t work.

if no way, I have to use two each ESP32.

**Comment by board707** [comment]
>  Both sides of the screen will display the exact same content, maybe there is a simpler way to display the same content on both sides,

Make two 160x80 screens and connect both in parallel to all pins

**Comment by metingul** [comment]
Problem solved, the esp support 160X160 pixels, I tried define PIXEL_COLOUR_DEPTH_BITS as 4 on sketch but did not affected, I modify directly on library file as 4 and screen appered.

how can I decrease color deepth on sketch?
